Signal/src/ViewControllers/ConversationView/CV/CVComponents/CVComponentMessage.swift

```diff
@@ -2079,11 +2079,13 @@ public class CVComponentMessage: CVComponentBase, CVRootComponent {
                                                            renderItem: renderItem,
                                                            messageSwipeActionState: messageSwipeActionState) {
             return panHandler
+        } else if componentView.contentViewSwipeToReplyWrapper.containsGestureLocation(sender) {
```

Comment:
There's probably a better way to do this like maybe installing the gesture recognizer on the bubble itself or something. That's a larger change than I'm wanting to make so I think this is a pretty good heuristic.

The contentViewSwipeToReplyWrapper appears to contain all of the message content that's translated in the reply swipe. I think it makes sense to use that as the gesture target? Anything outside of that area should not be recognized.